src/ekobox/evol_pdf.py

```diff
@@ -56,7 +57,7 @@ def evolve_pdfs(
     else:
         if store_path is None:
             raise ValueError("'store_path' required if 'path' is not provided.")
-        eko.solve(theory_card, operators_card, path=store_path)
+        managed.solve(theory_card, operators_card, path=store_path)
```

Comment:
Actually, this is a separate feature from the one described by the PR.

Make sure to advertise it properly.

(Also because the `managed` runner was not intended to be always used, but just for quick calculations -> if it becomes the default, this information should be propagated)

Reply:
@alecandido is still right :upside_down_face: you need to document this

Reply:
actually, let's do the proper thing and just make two PRs

Reply:
#363